src/test/java/api/support/fixtures/OpeningHourExamples.java

```diff
@@ -0,0 +1,18 @@
+package api.support.fixtures;
+
+import org.folio.circulation.domain.OpeningHour;
+import org.joda.time.LocalTime;
+
+public class OpeningHourExamples {
+  public static OpeningHour allDay() {
+    return new OpeningHour(new LocalTime(0, 0), new LocalTime(23, 59));
+  }
+
+  public static OpeningHour beforeNoon() {
+    return new OpeningHour(new LocalTime(7, 0), new LocalTime(12, 0));
+  }
+
+  public static OpeningHour afterNoon() {
```

Comment:
is `after noon`  two words or one? If it is one, please capitalise it as one word.

Reply:
done